src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js

```diff
@@ -0,0 +1,456 @@
+      describe(`when "recallsSectionOpen" prop is ${testRecallsSectionOpen}`, () => {
+        it('should render accordion', () => {
+          render(
+            <RequestManagementSection
+              {...defaultTestProps}
+              recallsSectionOpen={testRecallsSectionOpen}
+              policy={{
+                loanable: true,
+              }}
+            />
+          );
+
+          expect(Accordion).toHaveBeenNthCalledWith(
+            accordionCallOrderByPlace.recallsSection,
+            expect.objectContaining({
+              id: 'recallsSection',
+              open: testRecallsSectionOpen,
+              onToggle: accordionOnToggleMock,
+              label: labelIds.recalls,
+            }), {}
+          );
+        });
+      });
+    });
+  });
+
+  describe('"holds section" accordion', () => {
+    [
+      false,
+      true,
+    ].forEach((testHoldsSectionOpen) => {
+      describe(`when "holdsSectionOpen" prop is ${testHoldsSectionOpen}`, () => {
+        it('should render accordion', () => {
+          render(
+            <RequestManagementSection
+              {...defaultTestProps}
+              holdsSectionOpen={testHoldsSectionOpen}
+              policy={{
+                loanable: true,
+              }}
+            />
+          );
+
+          expect(Accordion).toHaveBeenNthCalledWith(
+            accordionCallOrderByPlace.holdsSection,
+            expect.objectContaining({
+              id: 'holdsSection',
+              open: testHoldsSectionOpen,
+              onToggle: accordionOnToggleMock,
+              label: labelIds.holds,
+            }), {}
+          );
+        });
+      });
+    });
+  });
+
+  describe('"recall return interval" period', () => {
+    it('should render period', () => {
+      render(
+        <RequestManagementSection
+          {...defaultTestProps}
+          policy={{
+            loanable: true,
+          }}
+        />
+      );
+
+      expect(Period).toHaveBeenNthCalledWith(
+        periodCallOrderByPlace.recallReturnInterval,
+        {
+          fieldLabel: labelIds.recallReturnInterval,
+          inputValuePath: 'requestManagement.recalls.recallReturnInterval.duration',
+          selectValuePath: 'requestManagement.recalls.recallReturnInterval.intervalId',
+          intervalPeriods: intervarPeriodsExpectedValue,
+          changeFormValue: changeMock,
+        }, {}
+      );
+    });
+  });
+
+  describe('"minimum guaranteed loan" period', () => {
+    it('should render period', () => {
+      render(
+        <RequestManagementSection
+          {...defaultTestProps}
+          policy={{
+            loanable: true,
+          }}
+        />
+      );
+
+      expect(Period).toHaveBeenNthCalledWith(
+        periodCallOrderByPlace.minimumGuaranteedLoan,
+        {
+          fieldLabel: labelIds.minimumGuaranteedLoanPeriod,
+          inputValuePath: 'requestManagement.recalls.minimumGuaranteedLoanPeriod.duration',
```

Comment:
i think this one and next three props can be moced into separate constant value. you use them at least four times.

Reply:
Could you please move 
```
intervalPeriods: intervarPeriodsExpectedValue,
changeFormValue: changeMock,
```
to variable and reuse it across all file?

Reply:
done